src/cms/templates/pois/poi_form.html

```diff
@@ -129,6 +133,13 @@ <h1 class="heading">
                     {% render_field poi_translation_form.minor_edit id="minor_edit" %}
                     <label for="{{ poi_translation_form.minor_edit.id_for_label }}" class="text-s p-2">{{ poi_translation_form.minor_edit.label }}</label>
                     <div class="my-2 text-s text-gray-600">{{ poi_translation_form.minor_edit.help_text }}</div>
+                    
+                    <span class="block mb-2 mt-4 font-bold">{% trans 'Location on the map' %}</span>
+                    {% render_field poi_translation_form.location_not_on_map id="location_not_on_map" %}
+                    <label for="{{ poi_translation_form.location_not_on_map.id_for_label }}" class="text-s p-2">{{ poi_translation_form.location_not_on_map.label }}</label>
+                    <div class="my-2 text-s text-gray-600">{{ poi_translation_form.location_not_on_map.help_text }}</div>
+
+
```

Comment:
Please place this field in the right sidebar because influences the POI object and not just one POITranslation.